docs/sample/dubbo.md

```diff
@@ -1,11 +1,13 @@
 # Dubbo
 
-Base on sample/dubbogo.
-
-* [dubbo-query](dubbogo-query.md)
-* [dubbo-body](dubbogo-body.md)
+* [dubbo-query](dubbo-query.md)
+* [dubbo-body](dubbo-body.md)
+* [dubbo-uri](dubbo-uri.md)
+* [dubbo-mix](dubbo-mix.md)
 * [dubbo-universality](dubbo-universality.md)
 * [dubbo-error](dubbo-error.md)
 * [dubbo-incomplete](dubbo-incomplete.md)
```

Comment:
![image](https://user-images.githubusercontent.com/12484497/102953512-70d71500-450c-11eb-8b90-126c8812ed77.png)
add more comments for every doc, like "dubbo-query: ....."

Reply:
only name add ... ?